test/functionalTest/harnessFunctions.sh

```diff
@@ -109,33 +109,39 @@ function dbInit()
   fi
   
   db=$host:$port/$1
+  
+  if [ $# == 2 ]
+  then 
+    db=$db$2
```

Comment:
Is this really work? I mean, if I use the function with: `dbInit CB t1`, then db=CB and $2=t1. Thus, the final DB  name will be `ftestt1` (i.e. missing the `-`).

Maybe it should be?

```
db=$db-$2
```